client.go

```diff
@@ -147,6 +148,9 @@ func (c *client) ThresholdProfiles() endpoints.ThresholdProfiles {
 func (c *client) UserGroups() endpoints.UserGroups {
 	return endpoints.NewUserGroups(c.restClient)
 }
+func (c *client) Users() endpoints.Users {
```

Comment:
Please add an empty line before this method and add a doc comment to be consistent with the other methods. Suggestion:

```
// Users implements Client.
```